File: uniffi_bindgen/src/bindings/dart/templates/Int8Helper.py
class _UniffiConverterInt8(_UniffiConverterPrimitiveInt) {
    CLASS_NAME = "i8"
    VALUE_MIN = -2**7
    VALUE_MAX = 2**7

    static read(buf) {
        return buf.read_i8()
    }

    static write_unchecked(value, buf) {
        buf.write_i8(value)
    }
}

File: uniffi_bindgen/src/bindings/dart/templates/Int8Helper.dart
class _UniffiConverterInt8 extends _UniffiConverterPrimitiveInt {
    int read(_UniffiRustBufferBuilder buf) {
        return buf.data.cast<Int8>().value;
    }

    _UniffiRustBufferBuilder write(value) {
        Pointer<_UniffiRustBuffer> _rustBuffer = calloc<_UniffiRustBuffer >();
        return _rustBuffer.ref.buffer;
        //buf.data.cast<Int64> = value;
    }
}
